packages/core/src/oot/ocarina.c

```diff
@@ -145,8 +167,277 @@ void Ocarina_HandleLastPlayedSong(GameState_Play* play, Actor_Player* player, s1
         {
             PlayerDisplayTextBox(play, 0x88d, NULL);
             changeAgeMessage(play, !gSaveContext.save.age);
-            gInCustomSong = 2;
+            sInCustomSong = CUSTOM_SONG_TIME;
+        }
+        break;
+    }
+}
+
+const char* soarNames[10] = {
```

Comment:
Static + prefix with `k` + no need for explicit size

Reply:
Ok, but why `k`?

Reply:
Common convention for const globals

Reply:
But it's not global it's static. should it be `sSoarNames`?